src/api/APIFactory.js

```diff
@@ -228,6 +238,15 @@ class APIFactory {
         this.recentsAPI = new RecentsAPI(this.options);
         return this.recentsAPI;
     }
+
+    /**
+     * API for versions
+     */
+    getVersionsAPI(): VersionsAPI {
```

Comment:
We may have to extend/improve these get*API calls to optionally take a flag that prevents calling this.destroy(). Until now it wasn't really needed because most API calls were not really called in parallel. Called one at a time. But versions will be called in parallel to say the File api call. So calling destroy will destroy the other. Likewise I didn't want two API calls to muck with the same cached object, but in versions case its a diff cache object, so not a big deal from that perspective.

Either that, or the call to getVersionAPI() should come in the fetch success callback of the file, so that its a serial call and not parallel.